### DISCUSSION TITLE
Quarkus 2.10.3.Final released - Fixes CVE-2022-2466

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#26820] [Quarkus Blog/Website]
# Quarkus 2.10.3.Final released - Fixes CVE-2022-2466

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-2-10-3-final-released/

**Comment by yuxblank** [comment]
Nice!
Glad to have reported this and see it already fixed in matter of days ;)
